front/inventory.php

```diff
@@ -69,5 +79,7 @@
     foreach ($headers as $key => $value) {
         header(sprintf('%1$s: %2$s', $key, $value));
     }
-    echo $inventory_request->getResponse();
+    if ($inventory_request->getHttpResponseCode() != 405) {
```

Comment:
I really do not understand why we should have a specific case for a `405`.

Reply:
Error 405 is to answer GET method is just not supported on `/front/inventory.php`. In that case, it is just not necessary to return any content. This is just to make glpi a little more green

Reply:
The c6ea1d2 commit remove this check and better handles the case in getResponse() by returning empty content if not content has been set.